util/BUILD

```diff
@@ -94,3 +94,9 @@ py_binary(
         requirement("pluralizer"),
     ],
 )
+
+py_binary(
+    name = "fusesoc_build",
+    srcs = ["fusesoc_build.py"],
+    deps = all_requirements,
```

Comment:
Sadly, have to grab everything, since it can launch any other Python script in the repo.